htdocs/compta/paiement/card.php

```diff
@@ -308,6 +312,22 @@
 // Amount
 print '<tr><td>'.$langs->trans('Amount').'</td><td>'.price($object->amount,'',$langs,0,-1,-1,$conf->currency).'</td></tr>';
 
+// External link
+if (!empty($object->ext_payment_id) && !empty($object->ext_payment_site) ) { 
+
+$stripe=new Stripe($db);
+$stripeacc = $stripe->getStripeAccount($object->ext_payment_site);
+
+if (!empty($stripeacc)) $connect=$stripeacc.'/';	
+  	$url='https://dashboard.stripe.com/'.$connect.'test/payments/'.$object->ext_payment_id;
+			if ($object->payment_site == StripeLive)
```

Comment:
You mean ->ext_payment_site here i think ?

Reply:
@eldy  yes, i fix it immediately